bootrom/Makefile

```diff
@@ -0,0 +1,15 @@
+bootrom_src = default-bootrom.S
+bootrom_ld = bootrom-link.ld
+bootrom_elf = bootrom.elf
+bootrom_hex = bootrom.hex
+bootrom_depth = 16
+
+GCC=riscv64-unknown-elf-gcc
+ELF2HEX=elf2hex
+
+$(bootrom_hex): $(bootrom_elf)
+	$(ELF2HEX) 4 $(bootrom_depth) $(bootrom_elf) > $@
```

Comment:
Can you see if Tim's approach using objcopy and xxd (https://github.com/riscv/riscv-isa-sim/blob/master/debug_rom/Makefile) will work here?  I'd rather not increase the use of elf2hex since it's a nonstandard tool, and this Makefile already depends on binutils being installed.